components/skeleton/demo/skeleton-test-link.js

```diff
@@ -36,8 +36,9 @@ export class SkeletonTestLink extends SkeletonMixin(LitElement) {
 			'd2l-link-small': this.type === 'small',
 			'd2l-skeletize': true
 		};
+		const widthSkeletonSize = `d2l-skeletize-${this.width}`;
```

Comment:
Interesting... so the new linting rules hated just referencing `this.something` in this case?

Reply:
My theory is that there is an error in some regex, any reference to `this` on the left-hand side of an assignment will break the rule. Which makes sense to prevent assigning properties on `render()`, but not so much when using it as an object key.